src/components/account/fission.ts

```diff
@@ -19,6 +20,21 @@ export type Annex = {
   requestVerificationCode: (
     formValues: Record<string, string>
   ) => Promise<{ requested: true } | { requested: false; reason: string }>
+
+  /**
+   * Create a progressive volume for a Fission account.
+   *
+   * This method can be used to load a file system before an account is registered.
+   * When you register an account, you can just keep using it as before.
```

Comment:
```suggestion
   * This method can be used to load a local-only file system before an account is registered.
   * When you register an account, the file system will sync with the Fission server, making it available through Fission IPFS nodes.
```

Possible rewrite to acknowledge the benefit of registering an account.